config.go

```diff
@@ -315,6 +316,7 @@ type Config struct {
 	ReadMacPath     string        `long:"readonlymacaroonpath" description:"Path to write the read-only macaroon for lnd's RPC and REST services if it doesn't exist"`
 	InvoiceMacPath  string        `long:"invoicemacaroonpath" description:"Path to the invoice-only macaroon for lnd's RPC and REST services if it doesn't exist"`
 	LogDir          string        `long:"logdir" description:"Directory to log output."`
+	LogCompressor   string        `long:"logcompressor" description:"Compression algorithm to use when rotating logs. Valid algorithms are: gzip and zstd."`
```

Comment:
Same comment here as in `btcd`: Could make use of the `choice` tag to validate values directly.